official/PianoDiQualifica/res/sections/revReports.tex

```diff
@@ -46,7 +46,61 @@ \subsection{Revisione di progettazione - RPmin}
+\end{itemize}
+
+\paragraph*{Norme di progetto}
+In sede di revisione è stata segnalata una mancanza di profondità nei contenuti, soprattutto nei confronti della mancanza delle procedure di automazione. Il \glossaryItem{team} ha quindi deciso di inserire maggiori dettagli riguardo a tali procedure, in particolare:
+\begin{itemize}
+\item aggiunta della descrizione dei comandi usati per la generazione automatica dell'elenco delle tabelle e delle figure;
+\item aggiunta della descrizione del processo per la generazione automatica del glossario dei termini;
+\item aggiunta della descrizione delle regole per la stesura della documentazione del codice e degli strumenti automatici per la generazione dei siti web ad essa associati;
+\item aggiunta della descrizione del tool di misurazione del codice.
+\end{itemize}
+Inoltre sono stati rimossi i riferimenti agli elenchi di figure e tabelle nell'indice del documento.
+
+\paragraph*{Analisi dei requisiti}
+Il documento è stato notevolmente migliorato nel periodo che ha portato alla RQ. I miglioramenti sono stati notati, ed ora il documento è conforme alle attese. Il team ha tuttavia notato un piccolo errore nella descrizione testuale del caso d'uso UC-S1. Tale errore è stato corretto.
+
+\paragraph*{Specifica tecnica}
+Anche in questo caso i miglioramenti del documento sono stati notati. Qualche dettaglio è però sfuggito alla revisione, e il team si è impegnato, in questo periodo, a correggere anche questi ultimi errori; in particolare:
+\begin{itemize}
+\item l'errore nel numero di versione è stato corretto;
+\item è stata migliorata la descrizione delle tecnologie utilizzate;
+\item sono state aggiunte le direzioni nel diagramma in figura 6;
+\item alcuni diagrammi sono stati ritoccati in modo da risultare più compatti e leggibili.
+\end{itemize}
+
+\paragraph*{Manuale amministratore}
+Essendo il manuale consegnato una bozza di quello definitivo il team si è impegnato a rendere più approfonditi e completi i contenuti. Inoltre sono state aggiunte le caratteristiche richieste per un documento formale.
+
+\paragraph*{Manuale utente}
+Essendo il manuale consegnato una bozza di quello definitivo il team si è impegnato a rendere più approfonditi e completi i contenuti. Inoltre sono state aggiunte le caratteristiche richieste per un documento formale.
+
+\paragraph*{Manuale sviluppatore}
+La consegna del manuale è stata insoddisfacente, in quanto è stato consegnato un confusionario archivio compresso contenente la documentazione di codice ed API. Questo ha portato alla mancanza di un punto di accesso ben preciso e reso difficile la consultazione. Inoltre sono stati evidenziati molti errori nella documentazione realizzata. Sono state quindi fatte le seguenti modifiche:
+\begin{itemize}
+\item corretti i doppi ''\'' negli URL (\textbf{U}niform \textbf{R}esource \textbf{L}ocator) delle API (\textbf{A}pplication \textbf{P}rogramming \textbf{I}nterface);
```

Comment:
@mdipirro attenzione, ti mancano gli escape. Guarda come viene compilato:
http://164.132.227.171/job/Documentation/1291/artifact/RA/Esterni/PianoDiQualifica.pdf (doc completo della tua compilazione - Jenkins da ora salva i pdf di ogni build che termina con successo, basta andare nel report della build e li vedi in prima pagina)
![screenshot from 2016-06-10 10 50 35](https://cloud.githubusercontent.com/assets/5928183/15959114/6342a5f4-2ef9-11e6-88f1-c070321551d5.png)